src/org/ods/orchestration/util/Project.groovy

```diff
@@ -1980,4 +2018,26 @@ class Project {
         }
     }
 
+    void addDefaults(String component) {
+        def rmURL = git.getOriginUrl()
+        def gitURL = this.getGitURLFromPath(this.steps.env.WORKSPACE, 'origin')
+        def actualUrl = gitURL.resolve("${this.key.toLowerCase()}-${component}.git").toString()
+        def repo = [
+            id: component,
+            data: [
+                openshift: [:],
+                documents: [:],
+            ],
+            include: false,
+            type: MROPipelineUtil.PipelineConfig.REPO_TYPE_ODS_CODE,
```

Comment:
We don't know the actual type. Do we need this field to be here? Otherwise, please, remove it.